packages/react-tooltip/src/components/Tooltip/Tooltip.types.ts

```diff
@@ -83,6 +115,9 @@ export type TooltipState = ComponentState<
   React.Ref<HTMLElement>,
   TooltipProps & {
     visible: boolean;
+    rendered: boolean;
+    arrowRef?: React.Ref<HTMLDivElement>;
+    arrowClassName?: string;
```

Comment:
nit: let's move this to an explicit type where we can document the internal state correctly